src/test/suite/fixtures/recorded/actions/bringVest.yml

```diff
@@ -12,17 +12,17 @@ command:
       modifier: {type: identity}
       insideOutsideType: inside
   extraArgs: []
-marks:
```

Comment:
Marks are now a property of the initial state, because they can and do change in response to a command.  Before this fact didn't matter because we only checked marks at the beginning.  We now test navigation map updating behaviour, so we also store how the marks look after we run a test